### ISSUE TITLE
What category should waydroid project fall into?

### ISSUE THREAD

**Issue by akimdi** [natpen/awesome-wayland#38]
[waydro.id](https://waydro.id)
[https://github.com/waydroid/waydroid](https://github.com/waydroid/waydroid)

**Comment by akimdi** [comment]
@apiraino  it works on Wayland only 🙂️

**Comment by apiraino** [comment]
I'm dumb, it's written also in the docs :-) 
https://waydro.id/#howto

**Comment by natpen** [comment]
I agree this looks like a really cool project, and will happily add it! Thanks for sharing.

But oh boy... what category indeed. Maybe a new one called "Emulation"?
